public/js/templates/ideaListItem.tpl

```diff
@@ -0,0 +1,6 @@
+<a class="ideaLink" href="/ideas/<%= name %>">
```

Comment:
I used :name as a command line option in the AppRouter.js file routes code..

However, I have changed it to ideaName ... and it still seems to work the same way..

I am probably missing something..

Thanks for your help..